requirements/dev.txt

```diff
@@ -90,7 +90,7 @@ jinja2==3.1.2
     #   code-annotations
 lazy==1.5
     # via -r requirements/needle.txt
-lazy-object-proxy==1.7.1
+lazy-object-proxy==1.8.0
```

Comment:
Dropped support for `Python 3.6` in the new version https://python-lazy-object-proxy.readthedocs.io/en/latest/changelog.html#id1